SparkPost/2web2ui
# SA-1868: useSparkPostQuery error handling

### What Changed

- useSparkPostQuery error defaults to an empty object instead of undefined if request fails due to network
- Error state added to charts

### How To Test

- Go to report builder
- Block network request for `time-series` in browser
 - Verify error state renders correctly
 - Verify error does not trigger error boundary page and keeps report builder rendered
 - Verify error message

### To Do
